tb_pulumi/cloudwatch.py

```diff
@@ -132,7 +132,7 @@ def __init__(
                 f'{self.name}-5xx',
                 name=f'{self.project.name_prefix}-5xx',
                 comparison_operator='GreaterThanOrEqualToThreshold',
-                dimensions={'LoadBalancer': f'app/{outputs['res_name']}/{outputs['res_suffix']}'},
+                dimensions={'LoadBalancer': outputs['res_suffix']},
```

Comment:
Since we use this more than once would be a good idea to pop this to a variable?

Reply:
I will evaluate this when I get back to this code today. I may have overlooked something while working through this bug.

Reply:
I looked at this a little bit, and here's the reason this isn't a separate variable (which would be cleaner were it so simple, I agree):

The variable `outputs` doesn't exist outside of this lambda. The lambda itself can't be more than a single instruction. So if we want to rename this variable for clearer meaning, we also have to place this call to `aws.cloudwatch.MetricAlarm` inside a separate function which the lambda calls. We do this in some places (like when the Fargate service declares a task definition, which is a complex structure that benefits from having this function with a bunch of inputs). If this is just a cosmetic issue, I'm not sure that makes things any clearer. I think I'm going to skip this one unless you feel very strongly about it.